sdk/src/main/java/io/dapr/actors/client/ActorProxyBuilder.java

```diff
@@ -39,6 +40,21 @@ public ActorProxyBuilder withPort(int port) {
         return this;
     }
 
+    public ActorProxyBuilder withHost(String host) {
+        this.clientBuilder.withHost(host);
+        return this;
+    }
+
+    public ActorProxyBuilder withThreadPoolSize(int threadPoolSize) {
+        this.clientBuilder.withThreadPoolSize(threadPoolSize);
+        return this;
+    }
+
+    public ActorProxyBuilder withOkHttpClientBuilder(OkHttpClient.Builder okHttpClientBuilder) {
```

Comment:
Let's not expose OKHttp to the layer above.

Reply:
No exposing the Http builder to the above layer will force us to use a client with only default configurations, which could be not very secure.